packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -310,25 +330,50 @@ export class Stepper implements LocalizedComponent, T9nComponent {
     updateMessages(this, this.effectiveLocale);
   }
 
-  private itemMap = new Map<HTMLCalciteStepperItemElement, { position: number; content: Node[] }>();
+  /** keep track of the currently active item position */
+  @State() currentPosition: number;
 
-  /** list of sorted Stepper items */
-  private items: HTMLCalciteStepperItemElement[] = [];
+  @Watch("currentPosition")
+  handlePositionChange(): void {
+    requestAnimationFrame(() => {
+      this.determineActiveStepper(true);
+    });
+  }
+
+  @State() elWidth: number;
+
+  @Watch("elWidth")
+  handleElWidthChange(): void {
+    requestAnimationFrame(() => {
+      this.determineActiveStepper();
+    });
+  }
 
   /** list of enabled Stepper items */
   private enabledItems: HTMLCalciteStepperItemElement[] = [];
 
-  /** keep track of the currently active item position */
-  private currentPosition: number;
+  private itemMap = new Map<HTMLCalciteStepperItemElement, { position: number; content: Node[] }>();
+
+  /** list of sorted Stepper items */
```

Comment:
i am thinking to remove the comment and keep the name `items`.  The reason why i added this earlier is to differentiate between `itemMap` and `items`. Looks like we can remove `itemMap` since it is not used anywhere else. Thoughts?